Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.py

```diff
@@ -1801,6 +1844,7 @@ def main() -> None:
 
         'google-drive-drives-list': drives_list_command,
         'google-drive-drive-get': drive_get_command,
+        'google-drive-drive-delete': drive_delete_command,
```

Comment:
The drive-create command is `google-drive-create`. Can they be standardized?
e.g. one of these options:

1. `google-drive-create` and `google-drive-delete`
2. `google-drive-drive-create` and `google-drive-drive-delete` (deprecate `google-drive-create` and call this method.)
3. Use a different naming scheme? (Can't think of any off the top of my head.)

Reply:
I decided to deprecate the `google-drive-create` command and replace it with `google-drive-drive-create` command to keep the naming standards.